### PR TITLE
First draft of the release post

### PR DESCRIPTION
**Pull request by samoht**
it's short and not very exciting. But at least it's there.

### CODE REVIEW

**Comment by samoht** [comment]
I should probably add a sentence about the self-hosted https://mirage.io website if I manage to get it on time...

File: tmpl/blog/announcing-mirage-25-release.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+Last summer, we introduced [Mirage
+v2.0](/blog/announcing-mirage-20-release): that release improved
+portability of unikernels by supporting [ARM
+device](/blog/introducing-xen-minios-arm); introduced a new way of
+coordinate distributed clusters of unikernels using Irmin, a prototype of
```
Comment (avsm) [inline tmpl/blog/announcing-mirage-25-release.md]: s/coordinate/coordinating/

File: tmpl/blog/announcing-mirage-25-release.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+[Git-like distributed, branchable storage](/blog/introducing-irmin); and
+improved security with the separate release of OCaml-TLS, the
+[transport layer security (TLS) in pure
+OCaml](/blog/introducing-ocaml-tls).
+
+Today we announce the new release of MirageOS v2.5, which incorporates
+numerous bug-fixes, major stability improvements (especially in the
```
Comment (avsm) [inline tmpl/blog/announcing-mirage-25-release.md]: href to the changes url here?

File: tmpl/blog/announcing-mirage-25-release.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+of the [Piñata](http://ownme.ipredator.se) security bounty, a new
+workflow to build secure static websites and the some insights about
+how we achieve getting our entropy right even in virtualised
+environment.
+
+The full list of changes is available on the [release](/releases) page and
+the breaking API changes now have their [own page](/wiki/breaking-changes).
+
+As usual, MirageOS 2.5.0 and the its ever-growing collection of
+libraries is packaged with the [OPAM](http://opam.ocaml.org) package
```
Comment (avsm) [inline tmpl/blog/announcing-mirage-25-release.md]: the URL is https://opam.ocaml.org, not http://

**Comment by amirmc** [comment]
notes to self:  Try to add info on the following while editing.

- [ ] Add page on security disclosure and contact address
- [ ] Add para on TLS Unix Tools
- [ ] Add call to action and links to what to try next (e.g. mirage-skeleton?)
- [ ] Add note about switch to https site and domain name

**Comment by amirmc** [comment]
To avoid confusion, I'll take a lock on this document (getting on a train soon).  I've pulled the post into a local branch and will make edits/additions there. Will push those up once I'm done and back on wifi.

My intent is to push this post live late this afternoon.

**Comment by amirmc** [comment]
This post is now in #345 (along with a bunch of other content). That PR is from a different branch on the same repo so anyone should be able to edit if needed.

I'm closing this PR so there isn't confusion.

**Pull request closed without merging**